cypress_tests/cypress/support/steps.ts

```diff
@@ -29,21 +29,21 @@ export const defineUserTypeAndTimeRestrictions = (): void => {
 
 export const selectFareType = (
     fareType: 'single' | 'period' | 'return' | 'flatFare' | 'multiOperator' | 'schoolService',
+    isScheme: boolean,
```

Comment:
FYI: if you wanted you could have defaulted this to false so you didn't have to update all the references 🙂

Reply:
Good to know, will keep in mind for next time